addons/mail/static/tests/discuss/core/attachment_panel_tests.js

```diff
@@ -14,8 +14,16 @@ QUnit.test("Empty attachment panel", async () => {
     const { openDiscuss } = await start();
     await openDiscuss(channelId);
     await click(".o-mail-Discuss-header button[title='Show Attachments']");
-    await contains(".o-mail-Discuss-inspector", {
-        text: "This channel doesn't have any attachments.",
+    await contains(".o-mail-ActionPanel", {
+        contains: ["p", { text: "This channel doesn't have any media attachments." }],
+    });
+    await click(".o-mail-Attachment-link");
+    await contains(".o-mail-ActionPanel", {
+        contains: ["p", { text: "This channel doesn't have any links." }],
+    });
+    await click(".o-mail-Attachment-file");
+    await contains(".o-mail-ActionPanel", {
+        contains: ["p", { text: "This channel doesn't have any file attachments." }],
```

Comment:
I don't feel like any need of checking `p` tag inside contains. Checking only text should be OK ?